code/starfield/starfield.cpp

```diff
@@ -2668,6 +2688,16 @@ void stars_delete_entry_FRED(int index, bool is_a_sun)
 	}
 }
 
+// Goober5000
+void stars_add_blank_background(bool creating_in_fred)
+{
+	Backgrounds.emplace_back();
+
+	// any background that is created will have correct angles, so be sure to set the flag
+	if (creating_in_fred)
```

Comment:
Why do we need this parameter? Why can't we just check `Fred_running` here?

Reply:
Because the code must run only when FRED is creating a new background.  It can't run when FRED is running but merely loading the file.